lib/rules/no-useless-escape.js

```diff
@@ -53,7 +55,8 @@ const REGEX_GENERAL_ESCAPES = new Set([
     "W",
     "x",
     "u"
-]);
+].concat(DIGITS));
```

Comment:
FWIW, I think this can be written as `new Set("\\0123456789bcdDfnrsStvwWxu")` since a string is iterable.